maro/simulator/scenarios/data_center/topologies/azure.2019.original/config.pm.size.yml

```diff
@@ -1,6 +1,13 @@
-N1:
-  Name: 'N1'
-  Amount: 100
-  CPU: 16
-  Memory: 112
-  GPU: 4
+PM_NODES:
+  P1:
+    Amount: 880
+    CPU: 16
+    Memory: 112
+    GPU: 4
```

Comment:
Currently, we do have GPU in our business logic. Why GPU here? configuration from the paper?

Reply:
It just means that we could add any features here. GPU is not used currently.

Reply:
comment it to disable it (``# GPU: 4``) now?
Or it's a bit misleading.